cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -294,5 +346,7 @@ spec:
     type: string
     description: The next turn up date-time
     JSONPath: .status.nextScaleUpTime
+{{ end -}}
```

Comment:
Should this be `{{- end }}`, like the others? Maybe its still valid -- or maybe it isn't causing problems right now because its the end of the file.